ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, triage
```

Comment:
Not sure that `triage` label is applicable to every repository. E.g. we don't use it for NeoGo.

Reply:
Let's have it, we don't have a proper set of states, so we have to stick to labels.